typescript/create-onchain-agent/templates/walletProvider/walletProvider.njk

```diff
@@ -0,0 +1,182 @@
+import { {{ baseClass }} } from "@coinbase/agentkit";
+import { Network } from "@coinbase/agentkit";
+{% if protocolFamily === "EVM" %}
+import { TransactionRequest, ReadContractParameters, ReadContractReturnType, Abi, ContractFunctionName, ContractFunctionArgs, Hex } from "viem";
+{% else %}
+import { Connection, PublicKey, RpcResponseAndContext, SignatureStatus, SignatureStatusConfig, VersionedTransaction, SignatureResult } from "@solana/web3.js";
+{% endif %}
```

Comment:
These lines are quite long in the generated file, let's pre-format them so they are broken up into multiple lines